lametro/views.py

```diff
@@ -80,9 +80,12 @@ def get_context_data(self, **kwargs):
         except:
             context['packet_url'] = None
 
-        # Create list of related board reports
+        # Create list of related board reports, ordered by descending last_action_date.
         if context['legislation'].related_bills.all():
-            context['related_bills'] = [LAMetroBill.objects.get(identifier=bill.related_bill_identifier) for bill in context['legislation'].related_bills.all()]
+            all_related_bills = context['legislation'].related_bills.all().values('related_bill_identifier')
+            related_bills = LAMetroBill.objects.filter(identifier__in=all_related_bills)
+            context['related_bills'] = sorted(related_bills, key=lambda bill: bill.last_action_date, reverse=True)
```

Comment:
this will work great, provided that all bills have a `last_action_date`. is it possible to have no last action date?